docs/reference/env.md

```diff
@@ -25,7 +25,7 @@ a particular machine.
     Options:
 
        --swarm	Display the Swarm config instead of the Docker daemon
-       --shell 	Force environment to be configured for a specified shell: [fish, cmd, powershell], default is sh/bash
+       --shell 	Force environment to be configured for a specified shell: [fish, cmd, powershell, tcsh], default is sh/bash
```

Comment:
I found it this way and I honestly wasn't sure why whomever added emacs didn't put it here; was thinking perhaps it wasn't supported code or something so didn't think it was my place to add it.